src/haddock/modules/topology/topocg/aa2cg.py

```diff
@@ -410,45 +431,49 @@ def determine_ss(structure, skipss, pdbf_path):
     Returns:
 
     """
+
     # calculate SS
     for model in structure:
+        tmp_file_name = f"../{output_path}/{pdbf_path.split('/')[-1][:-4]}_tmp.pdb"
+        create_file_with_cryst(pdbf_path, tmp_file_name)
+
         if skipss:
             continue
         else:
             try:
-                dssp = DSSP(model, pdbf_path)
+                p = subprocess.Popen(["dssp", tmp_file_name, "--output-format", "dssp"], stdout=subprocess.PIPE, stderr=subprocess.PIPE, text=True)
             except:  # TODO: think about making this exception more specific
                 # no secondary structure detected for this model
                 log.warning('SS could not be assigned, assigning code 1 to all residues')
                 continue
+        
+        dssp_raw, errors = p.communicate()
```

Comment:
This `communicate` needs to go just after the `subprocess.Popen`, else it can become unbound